recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  string: h{{ PKG_HASH }}_py{{ python | replace(".", "") }}_pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
+  number: 2
+  string: py{{ python | replace(".", "") }}h{{PKG_HASH}}__pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
```

Comment:
Here, after PKG_HASH, I see "__" double underscores. Also, was this a noarch package  as it has python version in its string.